autoload/codeium/command.vim

```diff
@@ -69,7 +69,7 @@ function! s:commands.Auth(...) abort
 
     echo 'Navigating to ' . url
     try
-      call system(browser . ' ' . "'" . url . "'")
+      call system(browser . ' ' . '"' . url . '"')
```

Comment:
What's the reason single quotes don't work on Windows?

Reply:
[Here](https://learn.microsoft.com/en-us/windows-server/administration/windows-commands/cmd#:~:text=If%20the%20directory%20path%20and%20files%20have%20spaces%20in%20their%20name%2C%20they%20must%20be%20enclosed%20in%20double%20quotation%20marks.%20For%20example%3A) it's mentioned that double quotes have to be used for `cmd` (the default shell for Vim on Windows). I couldn't found a Microsoft source, but [this stack overflow answer](https://stackoverflow.com/a/51090074) explains that single quotes are just regular characters for `cmd` almost always

Reply:
Makes sense, thanks.